danceradvent/public/articles/2015/3-medium-scale-part-1.pod

```diff
@@ -0,0 +1,100 @@
+=encoding utf8
+
+=head1 Medium-Scale Dancer, Part 1: Modules
+
+=head2 Or: How to Get Beyond Small-Scale in Dancer Application Design
+
+When you generate a Dancer app with C<dancer2 -a App>, you get a very
+nicely structured and useful starting point. As long as your app remains
+fairly small, you can work with this structure as-is clear through to
+completion. In that sense, the generated app is production-ready.
+
+The problem comes when you start getting to thousands of lines of Perl
+code, dozens of view files, and hundreds of routes, stretching the stock
+app design beyond its natural limits. This series of articles gives you
+one way to extend beyond this base.
+
+I say "one way" because Dancer is a policy-free web application
```

Comment:
Yes, you should, not only because it describes Dancer's current design slant, it contrasts it with that of so-called "opinionated" frameworks (Catalyst, Rails, Zope, etc.) which helps explain "Why Dancer?" Also, a battle flag helps keep the troops all pointed in the same direction.